src/db/migrations/0080_sparkling_gamma_corps.sql

```
@@ -0,0 +1,13 @@
+ALTER TABLE "retention_policies" ALTER COLUMN "database_id" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "alert_policy" ALTER COLUMN "database_id" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "storage_policy" ALTER COLUMN "database_id" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "projects" ADD COLUMN "backup_policy" text;--> statement-breakpoint
+ALTER TABLE "retention_policies" ADD COLUMN "project_id" uuid;--> statement-breakpoint
+ALTER TABLE "alert_policy" ADD COLUMN "project_id" uuid;--> statement-breakpoint
+ALTER TABLE "storage_policy" ADD COLUMN "project_id" uuid;--> statement-breakpoint
+ALTER TABLE "retention_policies" ADD CONSTRAINT "retention_policies_project_id_projects_id_fk" FOREIGN KEY ("project_id") REFERENCES "public"."projects"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)